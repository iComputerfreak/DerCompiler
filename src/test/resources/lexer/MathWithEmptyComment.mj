/* the next comment will be empty */
/**/
class math {
    public int add(int a, int b) {
        return a + b;
    }
    public int sub(int a, int b) {
        return a - b;
    }
}
