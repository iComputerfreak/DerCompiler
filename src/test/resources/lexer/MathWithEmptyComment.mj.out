class
identifier math
{
public
int
identifier add
(
int
identifier a
,
int
identifier b
)
{
return
identifier a
+
identifier b
;
}
public
int
identifier sub
(
int
identifier a
,
int
identifier b
)
{
return
identifier a
-
identifier b
;
}
}
EOF
